.gitignore

```
@@ -9,3 +9,5 @@ integration/out
 dist/
 **/.idea/
 **/tsconfig.tsbuildinfo
+.vimrc
+.nvimrc
```

consider to ignore settings file. So this file will not appear to git changes when you will change or remove `editor.formatOnSave` setting.
```suggestion
.nvimrc
.vscode/settings.json
```

---

Done.